interface/forms/functional_cognitive_status/new.php

```diff
@@ -46,8 +46,9 @@
         <style>
             @media only screen and (max-width: 768px) {
                 [class*="col-"] {
-                width: 100%;
-                text-align:left!Important;
+                    width: 100%;
+                    text-align:left!Important;
```

Comment:
We should try to find a way to work around this instead of modifying bootstrap code here as this code could be messed up further down the line.